split-llvm-extract.cpp

```diff
@@ -287,6 +289,8 @@ int main(int argc, char **argv)
 			}
 		}
 
+		// Create directory if it does not exist
+		std::filesystem::create_directory(outputDirectory.c_str());
```

Comment:
If I understand https://en.cppreference.com/w/cpp/filesystem/create_directory correctly, the directory will have `ogu+rwx` which is very permissive. Does that match what you see?

Reply:
We can set the permission to `o+rwx`. I see `rwxr-xr-x` (755).

Reply:
755 is OK. Just to check: if you delete the directory and recreate it you get 755? [755 would be sensible, but it's not quite what the docs say, so I'm checking!]

Reply:
Tried multiple times. Always 755.

Reply:
Thanks!